bokehjs/src/coffee/api/typings/models/tools.d.ts

```diff
@@ -78,19 +78,23 @@ declare namespace Bokeh {
     export var BoxSelectTool: { new(attributes?: IBoxSelectTool, options?: ModelOpts): BoxSelectTool };
     export interface BoxSelectTool extends SelectTool, IBoxSelectTool {}
     export interface IBoxSelectTool extends ISelectTool {
+        callback?: Callback | ((source: this) => void);
```

Comment:
If `callback` is shared by all subclasses of a class, then it should be defined on the superclass. However, we want custom documentation for `callaback` in each subclass. To do this, we actually have to define it in each subclass. I was hoping `Override` could help here, but there is only support for default values. Perhaps `Override` could be allowed to override `help` as well.